nixpkgs_review/review.py

```diff
@@ -367,6 +384,19 @@ def filter_packages(
         for regex in package_regexes:
             if regex.match(attr):
                 packages.add(attr)
+
+    if not packages:
+        packages = changed_packages
```

Comment:
I don't understand this line. Packages is initialized in line 370.

Reply:
Removed.

Reply:
This is actually required and now I remember why. If we do not explicitly add a package we want to build packages is empty and we can't discard anything.